tests/expectations/metrics/test_metrics_util.py

```diff
@@ -83,6 +89,23 @@ def unexpected_index_list_two_index_columns():
     ]
 
 
+@pytest.fixture
+def column_names_all_lowercase() -> list[str]:
+    return [
+        "artists",
+        "healthcare_workers",
+        "engineers",
+        "lawyers",
+        "scientists",
+    ]
+
+
+@pytest.fixture
+def column_names_all_uppercase(column_names_all_lowercase: list[str]) -> list[str]:
+    name: str
+    return [name.upper() for name in column_names_all_lowercase]
```

Comment:
Consider moving these fixtures down, right above the tests where they are used (e.g. line 298)

Reply:
@anthonyburdi I am happy to do that -- in the next pull request.  Should we adopt this as a general policy in test (as in: bunch up fixtures close to where they are used in the module?) -- we should probably discuss pros and cons first.  Thanks!